### PR TITLE
DOCSP-27700 Manage the Relation Model Refactor

### PR DESCRIPTION
**Pull request by ianf-mongodb**
## Description

Small updates and improvements to the Manage the Relation Model page. Also updated `Connect DB` -> `Connect to database` since the UI has changed and adjusted terminology `migration jobs` -> `sync jobs` for consistency.

## Staging

https://docs-mongodbcom-staging.corp.mongodb.com/docs-relational-migrator/docsworker-xlarge/DOCSP-27700/projects/manage-relational-connection/

## Jira

https://jira.mongodb.org/browse/DOCSP-27700

## Build

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=64d6476c67dbe91010dcc431

### CODE REVIEW

File: source/projects/manage-relational-connection.txt [hunk 1]
```diff
@@ -9,13 +9,13 @@ Manage the Relational Model
...
    :backlinks: none
-   :depth: 1
+   :depth: 2
    :class: singlecol
 
-You can use the :guilabel:`Manage` button on the :guilabel:`Schema model` pane to 
-make changes to the schemas or tables you want to include in 
+You can use the :guilabel:`Manage` button on the :guilabel:`Schema model` 
+pane to make changes to the schemas or tables you want to include in 
 your migration project. You can also connect to the source database
```
Comment (jmd-mongo) [inline source/projects/manage-relational-connection.txt:17]: [question]

Should this be "sync" to be consistent with the other changes, or would that inadvertently change the meaning here? Thanks!

File: source/projects/manage-relational-connection.txt [hunk 2]
```diff
@@ -28,40 +28,44 @@ the :guilabel:`Manage` button.
...
-#. Select either :ref:`Connect DB <connect-db>` or :ref:`Import DDL File  <import-ddl-file>`.
+#. Select either :ref:`Connect to database <connect-to-db>` or 
+   :ref:`Import DDL File  <import-ddl-file>`.
 
-.. _connect-db:
+.. _connect-to-db:
 
-Connect DB 
-~~~~~~~~~~~
+Connect to database
```
Comment (jmd-mongo) [inline source/projects/manage-relational-connection.txt:58]: [suggestion]

I think we want to capitalize "Database" here.

**Review by jmd-mongo — COMMENTED** [review]
Thanks for the updates, @ianf-mongodb! I've left a question and a suggestion for your consideration.

Thanks!
Joe

File: source/projects/manage-relational-connection.txt (hunk 2 above)
Comment (ianf-mongodb) [reply source/projects/manage-relational-connection.txt:58]: Unfortunately this is deliberate, the UI is string literal "Connect to database", if this changes I will update:

Comment (ianf-mongodb) [reply source/projects/manage-relational-connection.txt:58]: Screenshot 2023-08-11 at 3 34 22 PM

File: source/projects/manage-relational-connection.txt (hunk 1 above)
Comment (ianf-mongodb) [reply source/projects/manage-relational-connection.txt:17]: I think "migration project" is adequate here. The UI refers to the concept as a "sync job", I think the entire project makes sense as a "migration project" that utilizes "sync jobs" for the data transfer.

**Review by ianf-mongodb — COMMENTED** [review]
Thanks for the review @jmd-mongo I have responded to your feedback.

**Review by jmd-mongo — APPROVED** [review]
Thanks for the updates and explanation, @ianf-mongodb! This one LGTM.

Thanks,
Joe

**Pull request merged**